src/index.html

```diff
@@ -11,7 +11,146 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
+
+    <form 
+      action="#" 
+      method="post" 
+      name="personaleDate"
+    >
+      <fieldset class="block">
+        <legend class="information">Personal information</legend>
+        <label class="form-field">
+            Surname:
+          <input type="text" name="surname">
+        </label>
+
+        <label class="form-field">
+            Name:
+          <input type="text" name="surname">
+        </label>
+
+        <label class="form-field">
+            How old are You?
+          <input 
+            type="number" 
+            name="old" 
+            min="0" 
+            max="100" 
+            value="12">
```

Comment:
```suggestion
          <input 
            type="number" 
            name="old" 
            min="0" 
            max="100" 
            value="12"
          >
```
Закриваюча дужка повинна бути на новому рядку